### ISSUE TITLE
Volume normalization

### ISSUE THREAD

**Issue by Cu3PO42** [00SteinsGate00/MusicServer#2]
Volume normalization across all inputs would be useful to not have my ear drums blown up.

**Comment by j-holub** [comment]
Yes I have though about that as well. I have even looked into various methods to do this.

One is, to take the whole playlist and normalize it's volume. Won't work in this case. Another one is to normalize everything to a constant volume level. I think that should work.

I will look into how to do this properly.
